frontend/src/metabase/components/HeaderBar.jsx

```diff
@@ -1,53 +1,39 @@
+/* @flow */
 import React, { Component } from "react";
 
 import Input from "metabase/components/Input.jsx";
 import TitleAndDescription from "metabase/components/TitleAndDescription.jsx";
 
-import cx from "classnames";
-
-export default class Header extends Component {
+export default class HeaderBar extends Component {
```

Comment:
Actually that is not needed anymore in Flow, and for instance `defaultProps` is automatically inferred. This component could still have `props: {...}` [as described in React docs](https://flow.org/en/docs/frameworks/react/#toc-adding-types-for-react-component-props).